.azure/azure-pipelines.ci.yml

```diff
@@ -134,6 +134,20 @@ stages:
       tls: openssl
       extraBuildArgs: -DisableLogs -Generator Ninja -ToolchainFile cmake/toolchains/arm-pi-gnueabihf.toolchain.cmake
 
+# Unsupported currently.
+- stage: build_linux_clang
```

Comment:
I'd prefer to not make a separate stage...

Reply:
Then I have to modify the build-config-user.yml to have an extra string I can use to modify the job name. I can do that if you think that'd be cleaner.

Reply:
Yeah, I think I prefer that way.